src/cldr-data/bn-IN.ts

```diff
@@ -0,0 +1,244 @@
+/**
+ * Auto-generated Locale Data File for bn-IN.
+ * This is generated in the @box/cldr-data project.
```

Comment:
Can we import these files from `@box/cldr-data` directly rather than committing copies of them?

Reply:
@priyajeet wanted me to move them away from there and put them directly into BUIE (see the discussion in #moji on slack).